user/mongoStoreClient.go

```diff
@@ -105,6 +105,11 @@ func (c *Client) FindUsersByRole(ctx context.Context, role string) (results []*U
 	return c.findUsers(ctx, bson.M{"roles": role}, noUserMessage)
 }
 
+func (c *Client) FindUsersByEmailVerified(ctx context.Context, emailVerified bool) ([]*User, error) {
+	noUserMessage := fmt.Sprintf("no users found: query: emailVerified: %v", emailVerified)
+	return c.findUsers(ctx, bson.M{"emailVerified": emailVerified}, noUserMessage)
```

Comment:
The mongodb field is still `authenticated` sadly.
```suggestion
	return c.findUsers(ctx, bson.M{"authenticated": emailVerified}, noUserMessage)
```